nilearn/surface/surface.py

```diff
@@ -685,17 +687,16 @@ def load_surf_mesh(surf_mesh):
     surf_mesh : str or numpy.ndarray
         Either a file containing surface mesh geometry (valid formats
         are .gii .gii.gz or Freesurfer specific files such as .orig, .pial,
-        .sphere, .white, .inflated) or a list or tuple of two Numpy arrays,
-        the first containing the x-y-z coordinates of the mesh
-        vertices, the second containing the indices (into coords)
-        of the mesh faces.
+        .sphere, .white, .inflated) or two Numpy arrays organized in a list,
+        tuple or a namedtuple with the fields "coordinates" and "faces"
 
     Returns
     --------
-    [coords, faces] : List of two numpy.ndarray
-        The first containing the x-y-z coordinates of the mesh vertices,
-        the second containing the indices (into coords) of the mesh faces.
+    mesh : namedtuple
```

Comment:
namedtuple is not a type. mesh is a `nilearn.surface.surface.Mesh` (and also a `tuple`)